tests/drivers/nrfx_integration_test/Kconfig

```diff
@@ -12,7 +12,7 @@ config NRFX_ALL_DRIVERS
 	select NRFX_ADC		if HAS_HW_NRF_ADC
 	select NRFX_CLOCK	if HAS_HW_NRF_CLOCK && !CLOCK_CONTROL_MPSL
 	select NRFX_COMP	if HAS_HW_NRF_COMP
-	select NRFX_DPPI	if HAS_HW_NRF_DPPIC
+	select NRFX_GPPI	if HAS_HW_NRF_DPPI
```

Comment:
`HAS_HW_NRF_DPPI` does not exist.
I'd suggest to select `NRFX_GPPI` unconditionally and remove `select NRFX_PPI` as redundant then.

Reply:
done